internal/services/privatedns/parse/a_record.go

```diff
@@ -0,0 +1,75 @@
+package parse
```

Comment:
why are we modifying the private dns service?

Reply:
Should not be included here. Removed.